src/main/java/edu/kit/provideq/toolbox/mip/solvers/CplexMip.java

```diff
@@ -0,0 +1,66 @@
+package edu.kit.provideq.toolbox.mip.solvers;
+
+import edu.kit.provideq.toolbox.Solution;
+import edu.kit.provideq.toolbox.meta.SolvingProperties;
+import edu.kit.provideq.toolbox.meta.SubRoutineResolver;
+import edu.kit.provideq.toolbox.mip.MipConfiguration;
+import edu.kit.provideq.toolbox.process.ProcessRunner;
+import edu.kit.provideq.toolbox.process.PythonProcessRunner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Component;
+import reactor.core.publisher.Mono;
+
+/**
+ * {@link MipConfiguration#MIP} solver using IBM cplex.
+ */
+@Component
+public class CplexMip extends MipSolver {
+  private final String scriptPath;
+  private final String venv;
+
+  private final ApplicationContext context;
+
+  @Autowired
+  public CplexMip(
+      @Value("${path.cplex.mip}") String scriptPath,
+      @Value("${venv.cplex.mip}") String venv,
+      ApplicationContext context) {
+    this.scriptPath = scriptPath;
+    this.venv = venv;
+    this.context = context;
+  }
+
+  @Override
+  public String getName() {
+    return "IBM Cplex Solver for MIP";
```

Comment:
Rename according to [new standard](https://github.com/ProvideQ/toolbox-server/issues/123)

Algorithm (Technology)

Here probably just "Cplex" because the algorithm isn't clear, so we just mention the technology